### PR TITLE
Added type annotations

### LINKED ISSUE #7: Missing Type Annotations
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_simple_text_display.py:171
- [ ] adafruit_simple_text_display.py:180

### PR DESCRIPTION
**Pull request by isacben**
Hi!

This PR solves Missing Type Annotations #7 

Please let me know if you have any comments.

I tested with Adafruit Feather RP2040 and the ST7789 display.

Thank you!

### CODE REVIEW

File: adafruit_simple_text_display.py [hunk 1]
```diff
@@ -177,7 +183,7 @@ def __getitem__(self, item):
                 )
         return self._lines[item]
 
-    def add_text_line(self, color=(255, 255, 255)):
+    def add_text_line(self, color: Tuple[int, int, int] = (255, 255, 255)) -> label:
```
Comment (tekktrik) [inline adafruit_simple_text_display.py]: This should return `label.Label`

Comment (tekktrik) [reply adafruit_simple_text_display.py]: Also, goood catch on the input, looks like we should create an issue in `adafruit_display_text` to update the type annotation for `Label` to take `Tuple[int, int, int]` for `color` as well.

File: adafruit_simple_text_display.py [hunk 2]
```diff
@@ -168,7 +174,7 @@ def __init__(
         # Add first line
         self._lines.append(self.add_text_line(color=colors[0]))
 
-    def __getitem__(self, item):
+    def __getitem__(self, item: int) -> label:
```
Comment (tekktrik) [inline adafruit_simple_text_display.py]: This should also return `label.Label`

**Review by tekktrik — CHANGES_REQUESTED** [review]
A couple of changes, thanks for the submission!

**Comment by isacben** [comment]
Hi @tekktrik ,

Thank you for the comments! I have completed the changes.
I will have a look to the `adafruit_display_text` topic. :)

Thank you!

**Review by tekktrik — APPROVED** [review]
Nice work!  I've created https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/issues/175 as well.

**Pull request merged**